langserver/format.go

```diff
@@ -45,6 +48,15 @@ func (h *LangHandler) handleTextDocumentFormatting(ctx context.Context, conn jso
 	}
 
 	formatted := buf.Bytes()
+
+	if h.config.GoimportsEnabled {
+		imports.LocalPrefix = h.config.GoimportsLocalPrefix
+		formatted, err = imports.Process("", formatted, nil)
```

Comment:
So the file is actually formatted twice using `cfg.Print` (aka `gofmt`) and `imports.Process` (aka `goimports`). Seems unnecessary in my opinion...

In my opinion the code could look something like this:
```go
if h.config.GoimportsEnabled {
    // set LocalPrefix
    // imports.Process(...)
} else {
    // ast.SortImports(...)
    // cfg.FPrint(...)
}
```

Reply:
Yeah, I thought about this too. I have changed it now. It wasn't completely obvious to me how to do it though, but I ended up changing the `imports.Process()` call slightly, it seems to work.